sympl/_core/util.py

```diff
@@ -652,3 +653,52 @@ def get_final_shape(data_array, out_dims, direction_to_names):
                 np.product([len(data_array.coords[name])
                             for name in direction_to_names[direction]]))
     return final_shape
+
+
+def get_component_aliases(*args):
+    """
+    Returns aliases for variables in the properties of Components (e.g., Prognostics).
+
+    Notes
+    -----
+     -  If a variable shows up in the input_properties or diagnostic_properties
+        of two or more different Components, make sure they have the same 'alias'
+        keyword in all Components.
+     -  This function will also produces aliases for tendency diagnostics in
+        Prognostic objects that have been wrapped with a TendencyInDiagnosticsWrapper.
+        Tendency aliases are of the form "<variable_name>_tend_from_<tendency_source>",
+        where <variable_name> is the aliased (shorter) name of the model variable and
+        <tendency_source> is where the tendency come from (e.g., "dynamics" or "diffusion")
+
+    Args
+    ----
+    *args : Component
+        Components from which to fetch variable aliases from the input_properties,
+        output_properties, diagnostic_properties, and tendency_properties dictionaries
+
+    Returns
+    -------
+    aliases : dict
+        A dictionary with keys containing old variable names and values containing
+        new variable names
+    """
+
+    aliases = {}
+
+    # Update the aliases dict with the properties in each provided Component
+    for component in args:
+        # combine the input, output, diagnostic, and tendency variables into one dict
+        for prop_type in ['input_properties', 'output_properties',
+                          'diagnostic_properties', 'tendency_properties']:
+            if hasattr(component, prop_type):
+                component_properties = getattr(component, prop_type)
+                # save the alias (if there is one) for each variable
+                for varname, properties in component_properties.items():
+                    if 'alias' in properties.keys():
+                        aliases.update({varname: properties['alias']})
+
+    # If any variables were renamed, let's also shorten the "tendency" diagnostics
```

Comment:
You should remove this part - it's not something this function is designed to do based on its docstring or its name, and it's something a user can easily do themselves in one line outside of this function. Also, it's generally cleaner to say `aliases['tendency'] = 'tend'` :)